templates/Dockerfile-rpm.tmpl

```diff
@@ -1,6 +1,11 @@
 # vim:set ft=dockerfile:
 FROM %%os%%:%%release%%
 
+RUN [[ %%os%% != centos ]] || [[ %%release%% != 8 ]] || ( \
```

Comment:
centos-8 default repository mirrors are not working so I changed the mirror into a working one

Reply:
I guess you reverted this change later, right ?

Reply:
No, I fixed it in another commit in parallel and merged so PR does not show the difference now
https://github.com/citusdata/packaging/commit/eb05a7a46583ce5cb117fe4da33a823a2fbde000